src/NHibernate.Test/Async/Tools/hbm2ddl/SchemaUpdate/MigrationFixture.cs

```diff
@@ -25,18 +27,36 @@ namespace NHibernate.Test.Tools.hbm2ddl.SchemaUpdate
 	[TestFixture]
 	public class MigrationFixtureAsync
 	{
-		private async Task MigrateSchemaAsync(string resource1, string resource2, CancellationToken cancellationToken = default(CancellationToken))
+		private Configuration _configurationToDrop;
+		private FirebirdClientDriver _fireBirdDriver;
+		
+		[OneTimeSetUp]
+		public void OneTimeSetup()
 		{
-			Configuration v1cfg = TestConfigurationHelper.GetDefaultConfiguration();
-			var driverClass = ReflectHelper.ClassForName(v1cfg.GetProperty(Environment.ConnectionDriver));
+			var cfg = TestConfigurationHelper.GetDefaultConfiguration();
+			var driverClass = ReflectHelper.ClassForName(cfg.GetProperty(Environment.ConnectionDriver));
 			// Odbc is not supported by schema update: System.Data.Odbc.OdbcConnection.GetSchema("ForeignKeys") fails with an ArgumentException: ForeignKeys is undefined.
 			// It seems it would require its own DataBaseSchema, but this is bound to the dialect, not the driver.
 			if (typeof(OdbcDriver).IsAssignableFrom(driverClass))
 				Assert.Ignore("Test is not compatible with ODBC");
 
-			using (Stream stream = Assembly.GetExecutingAssembly().GetManifestResourceStream(resource1))
-				v1cfg.AddInputStream(stream);
-			await (new SchemaExport(v1cfg).ExecuteAsync(false, true, true, cancellationToken));
+			if (typeof(FirebirdClientDriver).IsAssignableFrom(driverClass))
+				_fireBirdDriver = new FirebirdClientDriver();
```

Comment:
It runs for every databases, but FireBird requires specific hacks to reduce the likeliness it may fail due to some schema locking it tends to do.